server/src/main/resources/application-local.yml

```diff
@@ -1,5 +1,23 @@
+micronaut:
+  security:
+    enabled: true
+    reject-not-found: false
+
 datasources:
   default:
     url: ${JDBC_URL:`jdbc:postgresql://localhost:5432/checkinsdb`}
     username: postgres
     password: 'postgres'
+
+credentials:
+  roles:
+    SUPER:
```

Comment:
If you want to test an endpoint that is secured you will need to use one of these users. The password will be the same as the username.

In the swagger-ui for example you will want to click authorize and enter username: ADMIN, password: ADMIN